recipes/jupyter-sshd-proxy/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jupyter-sshd-proxy" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/jupyter_sshd_proxy-{{ version }}.tar.gz
+  sha256: 61e0b058590f642cc6ef13b952826a7ad3127447cb986a840b8399eba93b1cfb
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - jupyter-server-proxy >=4.3.0
+    - python >=3.8
+
+test:
+  imports:
+    - jupyter_sshd_proxy
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/yuvipanda/jupyter-sshd-proxy
+  summary: Run sshd under jupyter
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - weiji14
+    - yuvipanda
```

Comment:
@yuvipanda, ok to add you as a conda-forge recipe maintainer here for `jupyter-sshd-proxy` (need an explicit yes/no answer)?

Reply:
Love it please do!

Reply:
Yes